client/app/components/button/RequestAccessButton.tsx

```diff
@@ -43,7 +42,13 @@ export default function RequestAccessButton({
       return;
     }
 
-    push(`/dashboard/select-operator/received/request-access/${operatorId}`);
+    // if user is requesting access as an admin, redirect to user operator form
+    if (isAdminRequest)
```

Comment:
I may be doing something wrong in the manul testing, because I keep ending up here: 
![image](https://github.com/bcgov/cas-registration/assets/77306764/a75259e5-dfa6-43f4-96b6-a15fc367b8ba)
And I don't know how to get here:
![image](https://github.com/bcgov/cas-registration/assets/77306764/4edc733c-084c-4add-b61d-ec241e82ff43)

I get redirected to that confirmation page after clicking the `request access as admin`, and if go back into the tile, I'm back at the select an operator page

Reply:
We can pair tomorrow to check this out.